src/chainbridgeConfig.ts

```diff
@@ -108,14 +108,14 @@ export const chainbridgeConfig: ChainbridgeConfig = {
       bridgeAddress: process.env.REACT_APP_ETHEREUM_BRIDGE_ADDRESS as string,
       erc20HandlerAddress: process.env
         .REACT_APP_ETHEREUM_ERC20_HANDLER_ADDRESS as string,
-      rpcUrl: process.env.REACT_APP_ETHEREUM_RPC_URL as string,
+      rpcUrl: "",
       type: "Ethereum",
       nativeTokenSymbol: "ETH",
       tokens: [
         {
           address: process.env.REACT_APP_ETHEREUM_ERC20_TOKEN_ADDRESS as string,
-          name: process.env.REACT_APP_ETHEREUM_ERC20_NAME as string,
-          symbol: process.env.REACT_APP_ETHEREUM_ERC20_SYMBOL as string,
+          name: "CERENETWORK",
```

Comment:
Do not forget DRY. Move value to constant.

Reply:
Updated.